src/components/Contacts/ContactsHeader/ContactsHeader.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactElement } from 'react';
+import ContactHeaders from './ContactsHeader';
+
+export default {
+  title: 'ContactHeaders',
+};
+
+export const Default = (): ReactElement => {
+  return <ContactHeaders />;
+};
+
+Default.story = {
+  parameters: {
+    chromatic: { delay: 1000 },
```

Comment:
We should eventually use the graphql mock provider here. I don't think it should do a real query. Not quite sure how to get it past the loading state. We'll have to do some testing.

We should spend some time making the testing wrappers consistent. They are wildly different between tests.

Reply:
Yeah I would like to use the graphql mock provider, I'm not sure how to best use it yet.  It sounds like you were having trouble making tests that required loading data also.  I agree that this is something we should work on.

Reply:
It works but you can't customize the mock data. https://github.com/CruGlobal/mpdx-react/pull/26/files#diff-3862096e13eaf47c09b8236fea6ae30abac092cae98e4d99c977fc11fea47c68R28-R41

I'll try and figure that out soon.